worker-policy-shared/src/main/java/com/github/cafdataprocessing/worker/policy/shared/MetadataValue.java

```diff
@@ -1 +1,37 @@
-/*
 * Copyright 2015-2017 Hewlett Packard Enterprise Development LP.
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *      http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */
package com.github.cafdataprocessing.worker.policy.shared;

/**
 * Holds a piece of metadata.
 */
public class MetadataValue {
    private String value;

    public MetadataValue(){}

    public MetadataValue(String value) {
        this.value = value;
    }

    public void setValue(String value) {
        this.value = value;
    }

    public String getValue(){
        return value;
    }
}

```

Comment:
The old version of the file appears to have been all on one line, my check-in has changed this format. Not sure if this is correct or not?